docs/types.rst

```diff
@@ -609,6 +646,21 @@ number of bytes, always use one of ``bytes1`` to ``bytes32`` because they are mu
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters!
 
+You can initialise a statically sized memory array inline using syntax such as ``string[] memory myarray = new string[](4);``. For example.
```

Comment:
This isn't quite complete, I'd like to match the text and the example.

Reply:
Perhaps better to move this into "Array literals / inline arrays"

Reply:
Done